nodl_python/nodl/errors.py

```diff
@@ -45,6 +49,8 @@ def __init__(self, message: str, element: etree._Element) -> None:
             f'Error parsing {element.tag} from {element.base}, line {element.sourceline}: '
             + message
         )
+        self.element = element
+        self.name = element.get('name', None)
```

Comment:
Not sure it's worth making this a part of the API if you can get it from the element.

Reply:
97698c7